reinforcement_learning/rlclientlib/utility/data_buffer.cc

```diff
@@ -3,50 +3,43 @@
 #include <iterator>
 #include <iostream>
 
-namespace reinforcement_learning { namespace utility {
-  data_buffer::data_buffer() = default;
+namespace reinforcement_learning {
+  namespace utility {
+    data_buffer::data_buffer() = default;
 
-  data_buffer::data_buffer(const translate_func& _translate) : translate(_translate) {}
```

Comment:
@ataymano Why did we add the translate function initially? Is it okay to remove?

Reply:
As far as I can answer, previous implementation only works for ds json. The translate function is to process "\n".

Reply:
Yes, it was required when we used \n as delimiters (to remove it from message bodies). Should be removed here.